.buildkite/commands/package_windows.ps1

```diff
@@ -0,0 +1,53 @@
+# Stop script execution when a non-terminating error occurs
+$ErrorActionPreference = "Stop"
+
+& "prepare_windows_host_for_node.ps1"
+
+# First try to get the env var from the process environment
+$windowsCertPassword = [System.Environment]::GetEnvironmentVariable('WINDOWS_CODE_SIGNING_CERT_PASSWORD', [System.EnvironmentVariableTarget]::Process)
+If ([string]::IsNullOrEmpty($windowsCertPassword)) {
```

Comment:
Gosh this Powershell syntax… 😅  I'm surprised there might not be a simpler way to read an env var (whatever the level in which it's defined, process or machine) 🤔